### ISSUE TITLE
Possiblity to avoid diff of Enumerable null vs empty

### ISSUE THREAD

**Issue by corbel88** [replaysMike/AnyDiff#5]
Hi,
Would it be possible to avoid the comparison from an Enumerable null vs empty?
Thanks

**Comment by replaysMike** [comment]
This is now included as of release version 1.0.61

**Comment by replaysMike** [comment]
For details on how to use, there is a new comparison option called `TreatEmptyListAndNullTheSame`. Usage is as follows:

```csharp
var provider = new DiffProvider();
var obj1 = new TestObject { IntCollection = new List<int> { } };
var obj2 = new TestObject { IntCollection = null };
var diff = provider.ComputeDiff<TestObject>(obj1, obj2, ComparisonOptions.All | ComparisonOptions.TreatEmptyListAndNullTheSame);
Assert.AreEqual(0, diff.Count);
```
